src/BenchmarkDotNet/Parameters/ParameterInstance.cs

```diff
@@ -55,8 +55,12 @@ public string ToDisplayText(CultureInfo cultureInfo)
             }
         }
 
-        public string ToDisplayText() => ToDisplayText(CultureInfo.CurrentCulture);
+        public string ToDisplayText(SummaryStyle summary)
+        {
+            return summary != null ? ToDisplayText(summary.CultureInfo, summary.MaxParameterColumnWidth) : ToDisplayText();
+        }
 
+        public string ToDisplayText() => ToDisplayText(cultureInfo, maxParameterColumnWidthFromConfig);
```

Comment:
I am sorry, my previous suggestion was incomplete

```suggestion
        public string ToDisplayText() => ToDisplayText(CultureInfo.CurrentCulture, maxParameterColumnWidthFromConfig);

```

Reply:
Sorry, I didn't notice. Thanks for fixing it!